app/controllers/products_controller.rb

```diff
@@ -84,7 +84,8 @@ def report
 
   # always responds with a zip file of (.qrda.zip files of (qrda category I documents))
   def patients
-    file = Cypress::CreateDownloadZip.create_total_test_zip(@product, 'qrda')
+    criteria_list = (@product.product_tests.checklist_tests.length>0) ? render_to_string(file: 'checklist_tests/print_criteria.html.erb', layout:'report') : nil
```

Comment:
You'll also need some spaces here between the `>` symbol and the elements next to it, and the `layout:` and `'report'` elements